src/cq_cam/tests/test_address.py

```diff
@@ -0,0 +1,81 @@
+import unittest
+
+import cadquery as cq
+
+from cq_cam.address import (
+    IJK,
+    XYZ,
+    ArcXAxis,
+    ArcYAxis,
+    ArcZAxis,
+    DwellTime,
+    Feed,
+    Speed,
+    ToolLengthOffset,
+    ToolNumber,
+    ToolRadiusOffset,
+    XAxis,
+    YAxis,
+    ZAxis,
+)
+
+
+class TestUtils(unittest.TestCase):
+    def test_x_axis(self):
+        gcode = f"{XAxis(10.0)}"
+        self.assertEqual("X10", gcode)
```

Comment:
Nothing against good old `unittest`, but consistency with other tests would use pytest style

```
class TestUtils:
    def test_x_axis(self):
        gcode = f"{XAxis(10.0)}"
        assert gcode == "X10"
```

(worth noting that the assert order is flipped in pytest style, actual == expected)